db/migrations/20240128161550_duplicate_vote_migration.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class DuplicateVoteMigration extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up(): void
+    {
+        $this->execute("
+            -- Alter table vote_logs
+            ALTER TABLE `vote_logs` ADD CONSTRAINT `unique_user_proposal_combination` UNIQUE (user_id, proposal_id);
```

Comment:
Is it possible this migration could fail due to having non-unique rows in the table?

Reply:
I manually cleared those rows previously where the bug was reported to me but I'll do a sweep before merging.